<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormUI</class>
 <widget class="QWidget" name="FormUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QPushButton" name="btn_ok">
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&amp;OK</string>
     </property>
     <property name="icon">
      <iconset resource="all.qrc">
       <normaloff>:/icons/ok</normaloff>:/icons/ok</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="btn_quit">
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&amp;Quit</string>
     </property>
     <property name="icon">
      <iconset resource="all.qrc">
       <normaloff>:/icons/quit</normaloff>:/icons/quit</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3" rowspan="5">
    <widget class="QLabel" name="os_image">
     <property name="toolTip">
      <string>Your platform icon</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_name">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Show QT about dialog</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Type your message:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="all.qrc"/>
 </resources>
 <connections/>
</ui>
